test/extensions/http/cache/file_system_http_cache/cache_file_fixed_block_test.cc

```diff
@@ -0,0 +1,96 @@
+#include "source/extensions/http/cache/file_system_http_cache/cache_file_fixed_block.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+namespace FileSystemHttpCache {
+
+class CacheFileFixedBlockTest : public ::testing::Test {
+public:
+  // Wrappers for private test-only functions.
+  void setFileId(CacheFileFixedBlock& block, uint32_t id) { block.setFileId(id); }
+  void setCacheVersionId(CacheFileFixedBlock& block, uint32_t id) { block.setCacheVersionId(id); }
+};
+
+namespace {
+
+TEST_F(CacheFileFixedBlockTest, InitializesToValid) {
+  CacheFileFixedBlock default_block;
+  EXPECT_TRUE(default_block.isValid());
+}
+
+TEST_F(CacheFileFixedBlockTest, GettersRecoverValuesThatWereSet) {
+  CacheFileFixedBlock block;
+  setFileId(block, 98765);
+  setCacheVersionId(block, 56789);
+  block.setBodySize(999999);
+  block.setHeadersSize(1234);
+  block.setTrailersSize(4321);
+  EXPECT_EQ(block.fileId(), 98765);
+  EXPECT_EQ(block.cacheVersionId(), 56789);
+  EXPECT_EQ(block.bodySize(), 999999);
+  EXPECT_EQ(block.headerSize(), 1234);
+  EXPECT_EQ(block.trailerSize(), 4321);
+}
+
+TEST_F(CacheFileFixedBlockTest, IsValidReturnsFalseOnBadFileId) {
+  CacheFileFixedBlock block;
+  setFileId(block, 98765);
+  EXPECT_FALSE(block.isValid());
```

Comment:
what is that makes this block invalid? Just not having all the fields set?

Maybe call this test UnsetFieldXXXIsInvalid, for XXXX with each individual field that needs to be set?

Reply:
... The test is named "IsValidReturnsFalseOnBadFileId"? [Most] unset fields are valid.

Reply:
My misunderstanding. To clarify for the reader: a CacheFileFixedBlock with unset fields is valid, but the FileId of 98765 is invalid for some reason? Maybe add a comment?

Reply:
Done (and for cacheVersionId).